profiles/serializers.py

```diff
@@ -18,6 +18,9 @@
 
 US_POSTAL_RE = re.compile(r"[0-9]{5}(-[0-9]{4}){0,1}")
 CA_POSTAL_RE = re.compile(r"[A-Z]\d[A-Z] \d[A-Z]\d$", flags=re.I)
+USER_NAME_RE = re.compile(
+    r"^(?![~!@&)(+:'.?/,`-]+)([^/^$#*=\[\]`%_;<>{}\"|]+)$", flags=re.I
```

Comment:
Could you document this regex using [verbose regular expressions](https://diveintopython3.problemsolving.io/regular-expressions.html#verbosere) so it's easy to understand what's going on here at a glance?

Reply:
Incorporated.